pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -124,6 +125,8 @@ type OperationExecutor interface {
 func NewOperationExecutor(
 	operationGenerator OperationGenerator) OperationExecutor {
 
+	util.RegisterMetrics()
```

Comment:
okay. i missed that. I would prefer that - we do this via `init` hook, because that wouldn't require `sync.Once` trick.  We will have to probably put metrics code in its own package for clarity in that case, since if a package contains multiple `init` hooks their execution can be wonky. 

Is there a reason - you didn't want to pull registration in that file itself?  let me know, if I am missing something. :D